### ISSUE TITLE
"self connect mode" setting has no effect.

### ISSUE THREAD

**Issue by Gooberpatrol66** [rncbc/qjackctl#134]
I have tried all of the options in "self-connect mode", and for all of them, pulseaudio JACK sink auto-connects to system output.

qjackctl 0.9.2
pulseaudio 13.0
jack2 1.9.17

**Comment by rncbc** [comment]
maybe because pulseaudio-jack module is no regular jack client or it is not itself what is issuing the proper connections...
maybe you should tacle this in the PA configuration or through a pactl command...

**Comment by Gooberpatrol66** [comment]
Just for context, I was able to disable self-connect requests for pulseaudio when I was using Carla.

**Comment by Gooberpatrol66** [comment]
Are there plans to fix this soon? I'm sick to death of pulse auto-connecting and deafening me.

**Comment by rncbc** [comment]
> Are there plans to fix this soon? I'm sick to death of pulse auto-connecting and deafening me.

i don't think so. as said, go for you may either go to your PA config and/or make a custom post-startup script to disconnect explicitly the PA's JACK source and/or sinks.
